roles/neutron-common/meta/main.yml

```diff
@@ -27,3 +27,10 @@ dependencies:
     dependent_packages: "{{ neutron.osp_sys_packages }}"
     rootwrap: True
     when: openstack_install_method == 'distro'
+  - role: openstack-distro
+    project_name: neutron-lbaas
+    dependent_packages: "{{ neutron.osp_sys_packages }}"
+    rootwrap: True
```

Comment:
yep, I agree. Acutally I just refer to others and we can double verify it later and clean up if needs be.